src/Compilers/CSharp/Portable/Binder/Binder_NameConflicts.cs

```diff
@@ -25,11 +25,12 @@ private static Location GetLocation(Symbol symbol)
             return locations.Length != 0 ? locations[0] : symbol.ContainingSymbol.Locations[0];
         }
 
-        internal void ValidateParameterNameConflicts(
+        internal void ValidateParameterNameConflicts<TParameterSymbol>(
```

Comment:
><TParameterSymbol> [](start = 52, length = 18)

Do parameters in function pointers have names? Why would this helper be affected by the feature?